### DISCUSSION TITLE
How to point github.io website to a custom domain with SSL installed?

### DISCUSSION THREAD

**Discussion by hyekangjoo** [community/community#22437] [Other Feature Feedback, Questions, & Ideas]
<p>I have one custom domain purchased, and I would like to use this custom domain for my <a href="http://github.io" rel="noopener nofollow ugc">github.io</a> website WITH SSL INSTALLED.</p>
<p>I get that you can do this by putting the custom domain website in CNAME in the root and putting the four IP addresses in A @ DNS, and I have done that, and it does let the website have my custom domain instead of .github.io.  However, what’s been killing me for the past two days is, I’ve been trying to install SSL – trying multiple means, including getting cloudflare (free), hostinger (paid) – but NONE of them worked.  So I would like to know how to get this fixed.</p>

**Comment by yoannchaudet** [comment]
<p>Hello and welcome here!</p>
<p>I see that your custom domain does not point to a GitHub IP address:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/8/682b99baad8b130c2fd66d52003e4cecb25bd478.png" data-download-href="/uploads/short-url/eRx4nixmtFR7unHkzp9V71AxGTC.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/14911070/181070899-74abee51-29c1-4238-ab5a-4d1ed5cad1c3.png" alt="image" data-base62-sha1="eRx4nixmtFR7unHkzp9V71AxGTC" width="690" height="362" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/8/682b99baad8b130c2fd66d52003e4cecb25bd478_2_690x362.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/8/682b99baad8b130c2fd66d52003e4cecb25bd478_2_1035x543.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/6/8/682b99baad8b130c2fd66d52003e4cecb25bd478.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/8/682b99baad8b130c2fd66d52003e4cecb25bd478_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1256×660 64.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>If you want GitHub to get a certificate on your behalf, your custom domain must point to our servers (as per <a href="https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain">our documentation</a>).</p>
<p>If after doing that you still have issues with your certificate, I am more than happy to investigate.</p>

**Comment by hyekangjoo** [comment]
<p>Fixed.  Thanks for the effort!</p>
